src/components/images/EditImage.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import LocalApi from "../../apis/LocalApi";
+import { Form, Button, Container } from "react-bootstrap";
+
+class EditImage extends Component {
+  state = {
+    selectedFile: null,
+    lot: 0,
+    unitNumber: "",
+    productDescription: ""
+  };
+
+  //When loading this page, retrive the data for this particular apartment from backend
+  componentDidMount() {
+    LocalApi.get(`${process.env.REACT_APP_BASEURL}/images/`)
+      .then(response => {
+        for (let i = 0; i < response.data.length; i++) {
+          if (response.data[i]._id === this.props.match.params.id) {
+            this.setState({
+              lot: response.data[i].lot,
+              unitNumber: response.data[i].unitNumber,
+              productDescription: response.data[i].productDescription
+            });
+          }
+        }
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  }
+
+  imageEditHandler = event => {
+    event.preventDefault();
+    const { lot, selectedFile, unitNumber, productDescription } = this.state;
+
+    LocalApi.post(`/images/edit/${this.props.match.params.id}`, {

```

Comment:
I just realised that we should have used PATCH method, so the call looked like this:
LocalApi.patch("/image/" + this.props.match.params.id) - that is more restful, see the table on https://en.wikipedia.org/wiki/Representational_state_transfer#Relationship_between_URI_and_HTTP_methods

Reply:
I know we should use 'patch', but so many tutorials I read online use 'post' so I'm so confused. Will try to change to patch to see if it works :)

Reply:
Use 'patch' and fixed.